.github/workflows/tagger.yml

```diff
@@ -17,32 +17,27 @@ jobs:
 
       - name: Check current version
         id: check_version
-        run: echo ::set-output name=VERSION::$(node -p "require('./package.json').version")
+        run: echo "VERSION=$(node -p "require('./packages/core/package.json').version")" >> $GITHUB_OUTPUT
```

Comment:
It seems that `"` is not escaped properly.

Reply:
That is not required to escape quotes in bash, I don't know why but it seems working.
![image](https://github.com/koishijs/koishi-plugin-booru/assets/19927330/7fc5df2b-dedc-4199-900f-e6fdce3356cf)

Reply:
Oh that's really amazing.